sources/packages/web/src/components/common/Reports.vue

```diff
@@ -78,6 +80,17 @@ export default defineComponent({
             PROGRAM_YEAR_DROPDOWN_KEY,
           );
         }
+        if (event.changed.value === PROGRAM_AND_OFFERING_STATUS) {
+          const institutionName = getFirstComponent(form, INSTITUTION_NAME);
```

Comment:
It should follow the same naming convention from the `programYearSelect`, which would be `institutionNameSelect`.